docs/Translation.md

```diff
@@ -0,0 +1,78 @@
+---
+description: BTCPay Translation features.
+---
+
+# Using BTCPay Translation Feature to Localize Your BTCPay Server instance
+
+
+BTCPay Server now includes a powerful translation feature that allows merchant administrators manage their instance in their preferred language. 
```

Comment:
```suggestion
Since version 2.0 BTCPay Server includes a translation feature that allows administrators to set up a default language for users accessing their instance.
```